definition/data/sheets/AuthorisationCaseField.json

```diff
@@ -189,6 +193,8 @@
   {"CaseTypeID": "MoneyClaimCase", "CaseFieldID": "paymentStatus", "UserRole": "letter-holder", "CRUD": "CRUD"},
   {"CaseTypeID": "MoneyClaimCase", "CaseFieldID": "paymentNextUrl", "UserRole": "letter-holder", "CRUD": "CRUD"},
   {"CaseTypeID": "MoneyClaimCase", "CaseFieldID": "paymentDateCreated", "UserRole": "letter-holder", "CRUD": "CRUD"},
+  {"CaseTypeID": "MoneyClaimCase", "CaseFieldID": "paymentTransactionId", "UserRole": "letter-holder", "CRUD": "CRUD"},
```

Comment:
should this be "CRUD" as we are merging to RC for prod?

Reply:
Ticket requires this to go into production as soon as possible